plugins/uberdev/shared/sdd-implementer-output-v1.md

```
@@ -0,0 +1,70 @@
+## SDD implementer output contract (v1)
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)